interface1.html

```diff
@@ -500,31 +500,26 @@ <h2>Options for prosodic tree generation (GEN function)</h2>
 						<td id="exhaustivityDetailOption1" style="display: none"><input type="checkbox" name="exhaustivityCats" value="i" checked="checked">&iota;'s children are &ge;&phiv;s&nbsp;&nbsp;</td>
 						<td id="exhaustivityDetailOption2" style="display: none"><input type="checkbox" name="exhaustivityCats" value="phi" checked="checked">&phiv;'s children are &ge;&omega;s&nbsp;&nbsp;</td>
 					</tr>
-					
 				</tbody></table>
 				<table class="spaceytable"><tbody>
 					<tr>
 						<td><input type="checkbox" name="genOptions" value="usesTones" id="annotatedWithTones">annotated with tones</td>
 						<td id="japaneseTones" style="display:none"><input type="radio" name="toneOptions" value="addJapaneseTones">Japanese</td>
 						<td id="irishTones" style="display:none"><input type="radio" name="toneOptions" value="addIrishTones_Elfner">Irish</td>
 					</tr>
-					<tr id="tones" style="display: none">
-						<td>
-							<input type="radio" name="toneOptions" value="addJapaneseTones">Japanese
-							<input type="radio" name="toneOptions" value="addIrishTones_Elfner">Irish
-						</td>
-					</tr>
 				</tbody></table>
-				<!--
-				<i>Annotate trees with tones for...</i><br/>
-				<table class="spaceytable"><tbody><tr>
-					<td><input type="radio" name="toneOptions" value="noTones" checked="checked">None</td>
-					<td><input type="radio" name="toneOptions" value="addJapaneseTones">Japanese</td>
-					<td><input type="radio" name="toneOptions" value="addIrishTones_Elfner">Irish</td>
-					
-				</tr></tbody></table>
-				<hr/>
-				-->
+				<span class="info"><span class="content" style="width: 500px">
```

Comment:
Upon further reflection, I think it would be better if this info was split into 2 info sections, one info button that goes with Japanese and another that goes with Irish. They should appear and disappear when the "annotate trees with tones" option is checked / unchecked.

Reply:
I can do that, but it really messes with the formatting once one of the info buttons is clicked. I'll send a commit with two separate info buttons so you can see what I mean.

Reply:
I've written a JS solution to the formatting issue trying to keep everything looking as similar as possible to the standard info button formatting and mechanics.